src/libs/actions/IOU.js

```diff
@@ -651,6 +651,7 @@ function buildOnyxDataForMoneyRequest(
  * @param {Object} [policy]
  * @param {Object} [policyTags]
  * @param {Object} [policyCategories]
+ * @param {Number} [moneyRequestReportID] - If user requests money from a composer on some money request report, we always add a request to that specific report.
```

Comment:
You must request from the Composer + button, not from the global create. Let me know if you prefer some other notation. Just want to clarify this works when you are inside some money request report and you click the + button inside the report composer.

Reply:
oh my goodness my brain absolutely didn't connect those dots 🤣 sorryyyy

Maaaaaaaybe this could be a tiny tiny bit clear with something like this:

```suggestion
 * @param {Number} [moneyRequestReportID] - If user requests money via the report composer on some money request report, we always add a request to that specific report.
```